src/components/views/rooms/RoomList.js

```diff
@@ -583,13 +583,18 @@ module.exports = React.createClass({
         }
     },
 
-    _makeGroupInviteTiles() {
+    _makeGroupInviteTiles(filter) {
         const ret = [];
+        const lcFilter = filter && filter.toLowerCase();
 
         const GroupInviteTile = sdk.getComponent('groups.GroupInviteTile');
         for (const group of MatrixClientPeg.get().getGroups()) {
-            if (group.myMembership !== 'invite') continue;
-            ret.push(<GroupInviteTile key={group.groupId} group={group} collapsed={this.props.collapsed} />);
+            const {groupId: id, name, myMembership: membership} = group;
```

Comment:
I suggest keeping the variable names here; `groupId` and `myMembership` are fine.

Reply:
I changed them when trying to squeeze it onto one line :P changed them back now :)

Reply:
ah :) thanks